liblangutil/SemVerHandler.cpp

```diff
@@ -116,7 +116,7 @@ bool SemVerMatchExpression::MatchComponent::matches(SemVerVersion const& _versio
 				cmp = static_cast<int>(_version.numbers[i] - version.numbers[i]);
 			}
 
-		if (cmp == 0 && !_version.prerelease.empty() && didCompare)
+		if (cmp == 0 && levelsPresent == 3 && !_version.prerelease.empty() && didCompare)
```

Comment:
Oh, I just noticed that `0.8` is apparently equivalent to `0.8.*` and not `0.8.0`. I did not expect that. Seems correct though, since the `semver` utility behaves the same way.